fvm/crypto/crypto.go

```diff
@@ -102,6 +132,8 @@ func RuntimeToCryptoHashingAlgorithm(s runtime.HashAlgorithm) hash.HashingAlgori
 		return hash.SHA2_384
 	case runtime.HashAlgorithmSHA3_384:
 		return hash.SHA3_384
+	case runtime.HashAlgorithmKMAC128_BLS_BLS12_381:
```

Comment:
I think it would be slightly cleaner to define fvm constants for hashing algorithms instead of relying on the flow-go/crypto ones. The reason is that we are referring to the customized KMAC supported by the FVM as the generic KMAC supported by the crypto library. 
This is an optional clean up though.